ext/include/opentelemetry/ext/http/client/curl/http_operation_curl.h

```diff
@@ -205,7 +205,7 @@ class HttpOperation
 
     /* wait for the socket to become ready for sending */
     sockfd_ = sockextr;
-    if (!WaitOnSocket(sockfd_, 0, http_conn_timeout_.count()) || is_aborted_)
+    if (!WaitOnSocket(sockfd_, 0, (long)http_conn_timeout_.count()) || is_aborted_)
```

Comment:
Style question:

Do we prefer C++ style casts?  Should this be `static_cast<long>` vs. `(long)`